src/hotspot/share/gc/shared/memAllocator.hpp

```diff
@@ -78,7 +78,7 @@ class MemAllocator: StackObj {
   // back to calling CollectedHeap::mem_allocate().
   HeapWord* mem_allocate(Allocation& allocation) const;
 
-  virtual MemRegion obj_memory_range(oop obj) const {
+  MemRegion obj_memory_range(oop obj) const {
```

Comment:
I don't see any usages of `obj_memory_range` anywhere in current sources. I think it was removed by [JDK-8171221](https://bugs.openjdk.org/browse/JDK-8171221). I think I'll just remove it in a separate PR: [JDK-8311249](https://bugs.openjdk.org/browse/JDK-8311249).